src/models/searchservice/response/directanswer/CustomFieldValueDA.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseFieldValueDirectAnswer } from './BaseFieldValueDirectAnswer';
+
+/**
+ * A {@link BaseFieldValueDirectAnswer} with a field type outside of {@link BuiltInFieldType}.
+ *
+ * @public
+ */
+export interface CustomFieldValueDA<T = unknown> extends BaseFieldValueDirectAnswer<T>{
```

Comment:
not sure if we want to keep `DirectAnswer` or shortern to `DA` for the custom and built-in names. It might be inconsistent but the whole thing seems a bit long and I didn't want to cut off any part of the name.

Reply:
If I had to pick between the two I think I slightly prefer the long name. Ideally there's some way to rejigger the models so we don't have long names...? Having a hard time thinking of a solution though

Reply:
I can update to the long name for now